### DISCUSSION TITLE
Choosing a Future Home for the REST API Standardization Repository

### DISCUSSION THREAD

**Discussion by aj-stein-nist** [usnistgov/OSCAL#1202] [Q&A]
In today's OSCAL REST API Standardization Meeting ([meeting notes here](https://hackmd.io/iQEXWmn_QRalExbrsZslsQ?both)), NIST staff and community participants discussed the creation of a central GitHub repository for this REST API standardization work.

There was good discussion but no final decision was made to either create a new repo under a given GitHub organization or under a specific GitHub username. We would like to solicit recommendations in comments on this post and have the community vote on their choice.

Please add one recommendation per top-level comment. If you wish to support the creation of a repository under a specific GitHub organization in an existing comment, please click the up arrow to vote.

We will review in a subsequent meeting and update on a final decision. Thanks again to all participants, in the meeting and in this discussion post. We appreciate your feedback.

**Comment by aj-stein-nist** [comment]
Please vote on this comment if you would like the repo to be created under the [`github/usnistgov` organization](https://github.com/usnistgov) on GitHub.

If not, please create a new comment to add a new recommendation or vote on alternative choices below.

**Reply by guyzyl** [reply]
Just so all the info is here, as raised in the meeting if the repo is created under `[github/usnistgov](https://github.com/usnistgov)` no one outside of NIST would be able to approve PRs therefore making this a community supported and not community owned project.

**Comment by xee5ch** [comment]
It seems there have been few offers outside the official NIST staff and the option of [github.com/usnistgov](https://github.com/usnistgov) org. With that in mind OSCAL Club (github.com/oscal-club) is here to offer a repo and GitHub infrastructure.

Why github.com/oscal-club?

- Allow NIST staff in the working group to be repo admins
- Allow non-NIST staff in the working group to be repo admins (where NIST staff indicated agency's GH policy and settings will not allow this)
- A friendly place for OSCAL that is not predominantly or majority controlled by NIST or a single commercial entity officially
- A friendly place that is an incubator that does not suggest official adoption by NIST or ownership of a single commercial entity until its ready
- A wonderful, tight-knit, and growing community :-)
